slapd.conf

```diff
@@ -0,0 +1,17 @@
+# See slapd.conf(5) for details on configuration options.
+include   /etc/ldap/schema/core.schema
+include   /etc/ldap/schema/cosine.schema
+include   /etc/ldap/schema/inetorgperson.schema
+include   /etc/ldap/schema/nis.schema
+#
+pidfile  /tmp/slapd/slapd.pid
+argsfile /tmp/slapd/slapd.args
+#
+modulepath /usr/lib/openldap
+#
+database  ldif
+directory /tmp/slapd
+#
+suffix    "dc=infra,dc=caasp,dc=local"
+rootdn    "cn=admin,dc=infra,dc=caasp,dc=local"
+rootpw    {SSHA}g34cKnFBLdgmFhZDajksMr5LwBGgN1Pm
```

Comment:
Should this be generated?

Reply:
I wasn't too concerned with generating it - this is used for tests in Travis CI only. Maybe I should rename this file to be more clear about that?

Reply:
Oh, didn't realize this is just for Travis... Maybe we can make it explicit somehow?